sql/migrations/warehouse/000003_add_upd_col_to_wh_schemas_and_uniq_constraint.sql.up.sql

```diff
@@ -0,0 +1,8 @@
+--
+-- wh_schemas
+--
+
+ALTER TABLE wh_schemas ADD COLUMN IF NOT EXISTS updated_at timestamp;
+
+-- adds a constraint to wh_schemas which allows us to do upsert(update/insert) on non primary columns
+ALTER TABLE wh_schemas ADD CONSTRAINT unique_wh_identifier UNIQUE("source_id","destination_id","namespace")
```

Comment:
@gane5hvarma 
Check for solutions to add constraint only if it does not exist. Otherwise, it might crash migrating to and fro between versions.
(something like this maybe ? https://stackoverflow.com/questions/6801919/postgres-add-constraint-if-it-doesnt-already-exist)

Reply:
added Exception handler which will ignore duplicate_object error